CMakeLists.txt

```diff
@@ -2,9 +2,9 @@ cmake_minimum_required(VERSION 3.13)
 
 # Build tests by default only if not a sub-project
 if(DEFINED PROJECT_NAME)
-    option(QUICR_BUILD_TESTS "Build tests for quicr" ON)
-else()
     option(QUICR_BUILD_TESTS "Build tests for quicr" OFF)
+else()
+    option(QUICR_BUILD_TESTS "Build tests for quicr" ON)
```

Comment:
Is this correct? PROJECT_NAME is only defined if this is a subproject?

Reply:
Are the ON/OFF backwards?

Reply:
If the project name is defined, it's a subproject (i.e., it's included inside of a different project). In that case, we want tests turned off.

Reply:
Ah, okay. Makes sense. Thanks!